frontend/packages/data-portal/app/components/Download/DownloadModal.tsx

```diff
@@ -106,12 +106,14 @@ function DownloadModalContent() {
         </div>
 
         <div>
-          <ButtonIcon
-            onClick={closeModal}
-            sdsSize="medium"
-            sdsType="tertiary"
-            sdsIcon="xMark"
-          />
+          <button onClick={closeModal} type="button">
```

Comment:
The `ButtonIcon` component has been merged into the `Button` component, you can use it like this:

```js
<Button
  sdsType="primary"
  sdsStyle="icon"
  icon="XMark"
  sdsSize="medium"
  onClick={() => {}}
/>
```